packages/fxa-content-server/app/styles/modules/_update_firefox.scss

```diff
@@ -0,0 +1,3 @@
+#fxa-cookies-disabled {
```

Comment:
Down below you use the id `fxa-update-firefox` and here you are using `fxa-cookies-disabled`. Expected?

Reply:
No, Corrected it.